### PR TITLE
Fix multi volcano scheduler doc error

### PR DESCRIPTION
**Pull request by Monokaix**
This pr：

1. replace k8s.io/dynamic-resource-allocation go mod
2. fix multi volcano scheduler doc error according to https://github.com/qiankunli/volcano/blob/3346870adc96d06572a2701ed9c1e0624b734e18/cmd/scheduler/app/options/options.go#L127

### CODE REVIEW

File: docs/design/multi-volcano-schedulers.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Multi schedulers
+# Multi volcano schedulers
```
Comment (wangyang0616) [inline docs/design/multi-volcano-schedulers.md:1]: Is multiple Volcano deployed in a cluster for scheduling?

**Comment by wangyang0616** [comment]
Please make CI happy. @Monokaix

File: docs/design/multi-volcano-schedulers.md (hunk 1 above)
Comment (Monokaix) [reply docs/design/multi-volcano-schedulers.md:1]: Yeah, and each volcano instance only cares about nodes that `node-selector` specified.

**Comment by Monokaix** [comment]
> Please make CI happy. @Monokaix

done for that.

**Comment by william-wang** [comment]
Thanks for your contribution :)

**Review by william-wang — APPROVED** [review]
/lgtm

**Pull request merged**